sdk/storage/Azure.Storage.DataMovement/KnownIssues.md

```diff
@@ -26,4 +26,40 @@ The Azure Storage REST service has various limitations as to the size of request
 
 ### Resume Behavior when Client Options are set
 
-When calling `*StorageResourceProvider.FromClient` with a Storage Client (e.g. `BlobBaseClient`, `ShareFileClient`) initialized with `*ClientOptions` (e.g `BlobClientOptions`, `ShareClientOptions`). It is NOT guaranteed that properties set within the `*ClientOptions` will be respected when resuming a transfer. 
\ No newline at end of file
+When calling `*StorageResourceProvider.FromClient` with a Storage Client (e.g. `BlobBaseClient`, `ShareFileClient`) initialized with `*ClientOptions` (e.g `BlobClientOptions`, `ShareClientOptions`). It is NOT guaranteed that properties set within the `*ClientOptions` will be respected when resuming a transfer.
+
+### Encryption Scope Support
+
+There is currently no support to specify an encryption scope for operations such as uploads, downloads, or copies.
+
+### Recursive Directory Support
+
+There is currently no support to handle operations that involve processing an entire directory, including all its subdirectories and files, automatically in a single command.
```

Comment:
I think I should have expanded on this in the github issue. I believe currently today we support transfers of entire directories (including it's child subdirectories and files) but we do not support disabling recursion.

What that means is that if someone only wanted to copy the files in a folder, and not the subdirectories in the folder, they cannot do that.